src/components/Heading/index.js

```diff
@@ -0,0 +1,47 @@
+import React, {PropTypes} from 'react'
+
+export const levels = ['1', '2', '3', '4', '5']
```

Comment:
This was mainly so I could use `oneOf(levels)` for the prop types:

```
Heading.propTypes = {
  children: PropTypes.node.isRequired,
  level: PropTypes.oneOf(levels).isRequired,
}
```

Probably a better way to do this. I've seen some people create `<H1>`, `<H2>` etc components that map to their html counterparts instead.